common/schemas.py

```diff
@@ -1,4 +1,17 @@
+from dataclasses import dataclass
+
 # Numbers represent the prefix of the id of the cards.
 # i.e. 26000037 is a Inferno Dragon, 27000007 is Elixir Collector, 28000000 is fireball
 CARD_TYPE_ID_PREFIX = {'26': 'troop', '27': 'building', '28': 'spell'}
 MAX_CARD_LEVEL = 14
+
+# Data Clases
+@dataclass
+class player_historical_activity:
+    tag: str
+    name: str
+    role: str
+    exp_level: int
+    fame_hist: list
+    boat_attacks_hist: list
```

Comment:
What's this for? Same with `deck_used_hist`. Aren't those fields out of scope for the requirements outlined in the ticket?